doc/source/getting-started/administration.rst

```diff
@@ -91,6 +91,12 @@ number of administrative reviews and approvals needed:
         approval from the direct manager, product line, and the chief
         technology officer. No product source code is allowed.
 
+.. note::
+
+   Business unit chief technology officer approval is needed for multi-physics and tools impacting
```

Comment:
I will say that having five adjectives before approval is really hard to digest! I'd suggest: For multi-physics and tools impacting several products, you must have approval from the chief technology officer for the Ansys business unit. For flagship-related projects, you must have general manager or vice president approval.

Is the "/" and "or" or an "and"--I went with "or" but may be incorrect. Also, my suggestion avoids using internal abrreviations, although I realize this topic is for Ansys employees. It's just best practice to avoid use of internal abbreviations in public-facing documentation.

Reply:
Sounds good, thanks @PipKat! Applying your suggestions right away. FYI - "or" was fine :)